xbuild/src/devices/imd.rs

```diff
@@ -132,9 +132,15 @@ impl IMobileDevice {
     }
 
     pub fn product_version(&self, device: &str) -> Result<(u32, u32)> {
+        fn _get_version(str: &str) -> Option<(&str, &str)> {
+            let mut splitter = str.split('.');
+
+            Some((splitter.next()?, splitter.next()?))
```

Comment:
What's wrong with calling `.split_once()` twice?

Reply:
> What's wrong with calling `.split_once()` twice?

if input str is “x.y”, split once first return (“x”, “y”), then “y”.split once panics.